e2e/designer/operationPanel.spec.ts

```diff
@@ -155,5 +155,23 @@ test.describe(
       await expect(page.locator('.msla-panel-layout-selected .msla-setting-section').first()).toBeVisible();
       await expect(page.locator('.msla-panel-layout-pinned .msla-peek')).toBeVisible();
     });
+
+    test('Should only show the panel info message when trigger type is request', async ({ page }) => {
```

Comment:
```suggestion
    test('Only show the panel info message when trigger type is request', async ({ page }) => {
```

Reply:
This one is a test name/description for us, not a customer facing string.